src/sentry/integrations/slack/webhooks/action.py

```diff
@@ -758,44 +787,42 @@ def get_action_option(cls, slack_request: SlackActionRequest) -> str | None:
         return action_option
 
     @classmethod
-    def get_action_list(cls, slack_request: SlackActionRequest) -> list[MessageAction]:
+    def get_action_list(cls, slack_request: SlackActionRequest) -> list[BlockKitMessageAction]:
         action_data = slack_request.data.get("actions")
-        if action_data:
```

Comment:
don't need this since we have ga'ed blockkit for months now and won't be dealing with non-blockkit interactions.